README.md

```diff
@@ -18,59 +29,59 @@ If you're new here, here are some things you should know:
+### Install Ruby
+You will need to first install the required ruby version specified in the `.ruby-version` file.  It is highly recommended you use a ruby version manager like [rbenv](https://github.com/rbenv/rbenv) or [asdf](https://asdf-vm.com/)
 
-For a general overview of the application, please see the [Application Overview](https://github.com/rubyforgood/human-essentials/wiki/Application-Overview) wiki article.
+GoRails has a very detail oriented guide for installing ruby on Ubuntu, Windows, and macOSX. You can check out that guide [here](https://gorails.com/setup/osx/12-monterey). **Follow only the Installing Ruby step, as our project setup differs**
 
-### Origins
+Once you've succesfully installed ruby. You may proceed to the next section!
 
+### Install Node
+You must install the version of node specified in `.nvmrc`. It is highly recommended to use [nvm](https://github.com/nvm-sh/nvm).
 
-This project took what we built for the [Portland Diaper Bank in 2016](https://github.com/rubyforgood/pdx_diaper) and turned it into a multitenant application, something that all essentials banks can use. We re-used models, code and other documentation where applicable as well as implemented new features and functionality requested by the prime stakeholder (PDXDB). We're super excited to have had Rachel Alston, the director of the Portland Diaper Bank, attending our event in 2017, providing guidance and giving us the best chance of success!
+You can follow the guide [here](https://heynode.com/tutorial/install-nodejs-locally-nvm/) for setting up node.
 
+*NOTE: It's possible that Node version 12 may cause you problems, see issue #751. Node 10 or 11 seem to be fine.*
 
-## Development
+### Install Postgres
+You must install postgres and run the database locally. Instructions differ depending on the operating system.
 
-### Installation Instructions
+Follow one of these guides to install postgres:
 
-The `installation.md` file ([https://github.com/rubyforgood/human-essentials/blob/main/installation.md](https://github.com/rubyforgood/human-essentials/blob/main/installation.md)) has detailed instructions for installation and configuration of an Ubuntu host to run this software. Although there is not a document for Mac OS, it may be helpful for that as well.
+###### MacOSX
+https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-your-ruby-on-rails-application-on-macos
```

Comment:
I ran the `bin/setup` script just after installing postgresql via homebrew and got the following error:

```ruby
ActiveRecord::DatabaseConnectionError: There is an issue connecting to your database with your username/password, username: postgres.

Caused by:
PG::ConnectionBad: connection to server at "::1", port 5432 failed: FATAL:  role "postgres" does not exist
```

Homebrew runs initdb after the postgresql installation. I had a role created for my user but no database with the name of my username so I couldn't use psql.
I had to create the `postgres` role manually using `createuser -P -d postgres`. The command appears in the digital ocean guide link but the fact that the role is named `postgres` and the password is `password` is specific to the application.

From https://github.com/rubyforgood/human-essentials/blob/main/config/database.yml

Reply:
@heyapricot do you think the removal of the `password` requirement locally for the DB will avoid this issue? If so, I'am not sure what the upside is for having a password locally.

Reply:
I'll do a clean setup with a modified version of config/database.yml (with the password removed) and see how that goes